### PR TITLE
Add purge_networks arg to docker driver

### PR DESCRIPTION
**Pull request by logan2211**
Users building containers in isolated docker networks may not want the
default docker bridge network attached.

This exposes the docker module's `purge_networks` argument in the driver configuration.

#### PR Type
- Feature Pull Request

### CODE REVIEW

**Comment by webknjaz** [comment]
Hi @logan2211, thanks for your contribution! Please rebase this PR on top of master.

**Comment by lwm** [comment]
Ping @logan2211!

**Comment by logan2211** [comment]
Hi, sorry for the delay. This is rebased now.

**Comment by lwm** [comment]
Added to agenda in https://github.com/ansible/community/issues/427. Please join if you have time!

File: molecule/driver/docker.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ class Docker(base.Base):
               - name: foo
               - name: bar
             network_mode: host
+            purge_networks: true
```
[the remark this answers is not in the document]
Comment (decentral1se) [reply molecule/driver/docker.py]: Yeah, I've seen that as well. Will try to stick to it in the future :+1:

**Comment by decentral1se** [comment]
:rocket:

**Pull request merged**